### ISSUE TITLE
Will the training code be made public in the future?

### ISSUE THREAD

**Issue by hklin363** [cszy98/PLACE#1]
Hello, thank you for your work. Will the training code be made public in the future?

**Comment by sky24h** [comment]
Hi, HongkLin, and anyone awaiting for the official release.

I'm not sure if you're still interested in the training code. 
Recently, I felt this paper could be useful for my research, so I created training code that has proved to work well with the COCO-stuff dataset.

Feel free to try my code instead:
[https://github.com/sky24h/PLACE
](https://github.com/sky24h/PLACE
)

**Comment by hklin363** [comment]
> Hi, HongkLin, and anyone awaiting for the official release.
> 
> I'm not sure if you're still interested in the training code. Recently, I felt this paper could be useful for my research, so I created training code that has proved to work well with the COCO-stuff dataset.
> 
> Feel free to try my code instead: [https://github.com/sky24h/PLACE ](https://github.com/sky24h/PLACE)

Thank you. Have you tried to compare the full training with the paper data? I want to know the gap in the reproduction process of hybrid training.

**Comment by sky24h** [comment]
Did you mean including this?
> We collect approximately 300k text-image pairs from OpenaImages [14] and Laion-5b [31] datasets as the Layout Free dataset. 

Sorry, I didn't consider the text part since my project only focuses on semantic layouts. 
Additionally, I'm uncertain whether this repository includes all the loss functions proposed in the paper. 
If not, implementing them could also be quite time-consuming, so I'm not planning to do it right now.

**Comment by hklin363** [comment]
OK，thank you！I just think this part might have a bigger impact on generalization. If it's just a closed set, the effect may be small.
















At 2024-07-25 03:34:54, "sky24h" ***@***.***> wrote:

Did you mean including this?

We collect approximately 300k text-image pairs from OpenaImages [14] and Laion-5b [31] datasets as the Layout Free dataset.

Sorry, I didn't consider the text part since my project only focuses on semantic layouts.
Additionally, I'm uncertain whether this repository includes all the loss functions proposed in the paper.
If not, implementing them could also be quite time-consuming, so I'm not planning to do it right now.

**Comment by hklin363** [comment]
> Hi, HongkLin, and anyone awaiting for the official release.
> 
> I'm not sure if you're still interested in the training code. Recently, I felt this paper could be useful for my research, so I created training code that has proved to work well with the COCO-stuff dataset.
> 
> Feel free to try my code instead: [https://github.com/sky24h/PLACE ](https://github.com/sky24h/PLACE)

May I ask what are the miou and FID of the COCO experiment you reproduced?
